src/alchemlyb/parsing/amber.py

```diff
@@ -0,0 +1,264 @@
+"""Parsers for extracting alchemical data from amber output files.
+Most of the file parsing part are inheriting from alchemical-analysis  
+Change the final format to pandas to be consistent with the alchemlyb format
+"""
+
+import os
+import re
+import logging 
+import pandas as pd
+import numpy as np
+
+from util import anyopen
```

Comment:
Needs to be 
```python
from .util import anyopen
```

Reply:
https://travis-ci.org/alchemistry/alchemlyb/jobs/301541570

Reply:
Oops, yes, fixed this